src/middleware/datasetTaskList.middleware.js

```
@@ -84,59 +97,22 @@
-
-const validateParams = validateQueryParams({
-  schema: v.object({
-    lpa: v.string(),
-    dataset: v.string()
-  })
-})
+/* eslint-disable-next-line no-return-assign */
+const emptyIssuesList = (req) => req.issues = []
 
 /* eslint-disable-next-line no-return-assign */
 const zeroEntityCount = (req) => req.entityCount = { entity_count: 0 }
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)